<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="Mail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>901</width>
     <height>481</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Email&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Email&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Gestion Client</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FirstName">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_LastName">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Day of birth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>     CIN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CIN">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;
</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>    E-Mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Email">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Type of Event</string>
     </property>
    </widget>
    <widget class="QComboBox" name="event">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Wedding</string>
     </property>
     <item>
      <property name="text">
       <string>Wedding</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Party</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Birthday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bachelor Party</string>
      </property>
     </item>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>531</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>Client </string>
     </property>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>330</y>
       <width>531</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Search</string>
     </property>
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CIN2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="CIN1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="FirstName1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="LastName1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit_FirstName</zorder>
    <zorder>label_2</zorder>
    <zorder>lineEdit_LastName</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>lineEdit_CIN</zorder>
    <zorder>label_5</zorder>
    <zorder>lineEdit_Email</zorder>
    <zorder>label_6</zorder>
    <zorder>event</zorder>
    <zorder>tableView</zorder>
    <zorder>groupBox_2</zorder>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Information</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>231</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QPushButton" name="sort">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_firstname">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_lastname">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_CIN">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Stat">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>591</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Excel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>390</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Export">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>390</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Email</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white
;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>851</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>attachment</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>761</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browse">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>130</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Send">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>861</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Content</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
